app/models/sir_trevor_rails/blocks/solr_documents_block.rb

```diff
@@ -24,7 +24,7 @@ def documents?
     end
 
     def text?
-      !text.blank?
+      !text.blank? and text != "<p><br></p>"
```

Comment:
Ah, I'm needing to do the exact same thing on the oEmbed block.  I'm using `#present?` instead of `!#blank?` though.  Would it be better to use `text.present? && text != '<p><br></p>'`?  I'll defer to and use what you decide.